### ISSUE TITLE
Do you plan a update to v5.7.0?

### ISSUE THREAD

**Issue by landrix** [emanuelbalea/MaterialDrawer#1]
Do you plan a update to v5.7.0?

thanks

**Comment by emanuelbalea** [comment]
Honestly didn't think anyone but me used this haha. Yeah I'll do it early next week.

**Comment by landrix** [comment]
Yes, i moving from android studio to xamarin and be happy to see your MaterialDrawer wrapper in nuget.

**Comment by landrix** [comment]
sorry, i try to update the lib by myself.

first the lib FastAdapter to FastAdapter-2.0.0\library\build\outputs\aar
which depends on buildTools 25.0.0

updating the FastAdapter.sln to nuget Xamarin.Android.Support.... 24.2.1 fails with an error.
i have no idea, what i need to do.
can you give me an hint?

**Comment by landrix** [comment]
ok, i should wait for Xamarin Android Support 25

**Comment by Drutol** [comment]
Hi! Anyone still alive? Can I hope for any updates or should I think about binding this myself?
